CHANGELOG.md

```diff
@@ -47,8 +47,10 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ## Unreleased
 
+
 ### New
 
+- **General**: Add formula for composing external metrics ([#2440](https://github.com/kedacore/keda/issues/2440))
```

Comment:
```suggestion
- **General**: Add support for formula based evaluation of metrics values ([#2440](https://github.com/kedacore/keda/issues/2440))
```

Reply:
Same here, let's add experimental in there